legal_imprint/legal_imprint_entity_template.md

```diff
@@ -0,0 +1,15 @@
+# Legal Imprint
+
+This file contains Legal Disclaimers for linking in Issues and Pull Requests.
```

Comment:
Suggesting to delete line 3 as it does not add value and rather limits the scope.

Constantin Hoferer, constantin.hoferer@daimler.com, Daimler AG on behalf of Daimler TSS GmbH.
[Imprint](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md).

Reply:
Okay, perfect. I will delete this line.